Scaling-Service-Example/README.md

```diff
@@ -0,0 +1,29 @@
+# Scaling Service Example Script
+
+This script allows consumers to manage the different resources otherwise referred to as "nodes" within their specific service instances. Consumers are able to specify the exact number of nodes they want running within a cell, and the script automatically starts and stops the nodes in order to meet their specifications. 
+
+This script can be used in conjunction with a chron job to start an appropriate workload during the morning and scale it back down at the end of the day. Using a Pay as You Go instance, the consumer would only pay 5% of the Pay as You Go while the guests were stopped. 
```

Comment:
Do you think that I should also have directions, or at least a link to a page on how to setup a cron job? Is that a good idea or doing too much?